plugins/node/opentelemetry-instrumentation-pg/test/pg-pool.test.ts

```diff
@@ -114,6 +114,10 @@ describe('pg-pool', () => {
   function create(config: PgInstrumentationConfig = {}) {
     instrumentation.setConfig(config);
     instrumentation.enable();
+
+    // Disable and enable the instrumentation to visit unwrap calls
+    instrumentation.disable();
```

Comment:
does this means that if a user adds a new instrumentation, they also need to disable and enabled?

Reply:
No, I added this to visit the `unwrap` calls in the tests to ensure they don't error. They were not visited in tests in the coverage report.